ibis/expr/operations/temporal.py

```diff
@@ -207,16 +207,6 @@ class DayOfWeekName(Unary):
     output_dtype = dt.string
 
 
-@public
-class DayOfWeekNode(Node):
-    arg = rlz.one_of([rlz.date, rlz.timestamp])
-
-    def to_expr(self):
```

Comment:
I wonder whether we have similar "unused" nodes left, `def to_expr()` should indicate a custom behaviour.